files/install-ptfe.sh

```diff
@@ -2,9 +2,6 @@
 
 set -e -u -o pipefail
 
-# Install pre-reqs
-apt-get update -y
-apt-get install -y jq chrony ipvsadm unzip wget
```

Comment:
do we really not need these anymore?

Reply:
They are included further down inside an OS check so they don't attempt to run against RHEL boxes.